pyproject.toml

```diff
@@ -47,6 +47,27 @@ content-type = "text/markdown"
 [project.urls]
 Homepage = "https://github.com/fake-useragent/fake-useragent"
 
+[tool.ruff]
+line-length = 142
+select = ["B", "C4", "C9", "E", "F", "I", "PL", "S", "SIM", "U", "W", "YTT"]
+ignore = ["B904", "C408", "PLW2901", "SIM105", "SIM108"]
+target-version = "py37"
```

Comment:
Since `isort` isn't supported anymore for Python 3.7. Should we move to Python 3.8 here as well maybe?

Reply:
(also maybe rebase, just to be sure)

Reply:
hm ok. Since 3.7 was giving issues during GitHub Actions, I removed it from the workflow and tox.